Description: Workaround for bug with systemd integration test
 Any systemd release after v259-rc2 should not require this patch anymore.
Author: Simon Kelley <simon@thekelleys.org.uk>
Bug: https://github.com/systemd/systemd/pull/40011
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1122015
Applied-Upstream: https://github.com/systemd/systemd/commit/e2fbcee
Reviewed-by: Sven Geuer <sge@debian.org>
Last-Update: 2025-12-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/dhcp-common.c
+++ b/src/dhcp-common.c
@@ -710,7 +710,7 @@
   { "posix-timezone", 100, OT_NAME }, /* RFC 4833, Sec. 2 */
   { "tzdb-timezone", 101, OT_NAME }, /* RFC 4833, Sec. 2 */
   { "ipv6-only", 108, 4 | OT_DEC },  /* RFC 8925 */
-  { "captive-portal", 114, OT_NAME },  /* RFC 8910 */
+  { "captive-portal", 114, 0 },  /* RFC 8910 */
   { "subnet-select", 118, OT_INTERNAL },
   { "domain-search", 119, OT_RFC1035_NAME },
   { "sip-server", 120, 0 },
@@ -752,7 +752,7 @@
   { "ntp-server", 56, 0 /* OT_ADDR_LIST | OT_RFC1035_NAME */ },
   { "bootfile-url", 59, OT_NAME },
   { "bootfile-param", 60, OT_CSTRING },
-  { "captive-portal", 103, OT_NAME },  /* RFC 8910 */
+  { "captive-portal", 103, 0 },  /* RFC 8910 */
   { NULL, 0, 0 }
 };
 #endif
